docs/docs/Get-Started/get-started-installation.md

```diff
@@ -3,170 +3,222 @@ title: Install Langflow
 slug: /get-started-installation
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 Langflow can be installed in two ways:
 
-* As a [Python package](#install-langflow-locally)
-* As a [standalone desktop application](#install-langflow-desktop)
+* As a [Python package](#install-and-run-langflow-oss) with Langflow OSS
+* As a [standalone desktop application](#install-langflow-desktop) with Langflow Desktop
 
-## Install Langflow locally
+## Langflow OSS Prerequisites
 
-Install Langflow locally with [uv (recommended)](https://docs.astral.sh/uv/getting-started/installation/), [pip](https://pypi.org/project/pip/), or [pipx](https://pipx.pypa.io/stable/installation/).
+- [Python 3.10 to 3.13](https://www.python.org/downloads/release/python-3100/)
+- [uv](https://docs.astral.sh/uv/getting-started/installation/) or [pip](https://pypi.org/project/pip/)
+- A virtual environment created with [uv](https://docs.astral.sh/uv/pip/environments) or [venv](https://docs.python.org/3/library/venv.html)
```

Comment:
This doesn't work as the first sub-section because it should be under the install and run OSS LF section, not directly under Install Langflow. Remove the heading and move the bulleted list between the heading "Install and run Langflow OSS" and step 1. In other words, the Install and run Langflow OSS topic should begin with "Before you install and run Langflow OSS, be sure you have the following items:"